docs/fud/fpga.md

```diff
@@ -0,0 +1,12 @@
+# FPGA
+
```

Comment:
Maybe add an overview paragraph here describing what this document is about? Specifically:

* What is the difference between simulation and actual FPGA execution?
* What toolchains/vendors do we currently support?
* What's the role of fud in this whole thing?

Reply:
Yep, I was missing a bit of detail since I just wanted to first get the PR started. I'll update such information soon.